frontend/app/src/components/history/transactions/TransactionContent.vue

```diff
@@ -495,7 +497,12 @@ const showDeleteConfirmation = () => {
 const { txEvmChains, getEvmChainName, getChain } = useSupportedChains();
 const txChains = useArrayMap(txEvmChains, x => x.id);
 
-onMounted(async () => refreshTransactions());
+onMounted(async () => {
+  await fetchData();
+  if (await refreshTransactions()) {
+    await fetchData();
+  }
```

Comment:
Shouldn't loading finishing cover this?https://github.com/rotki/rotki/blob/4637dcbd727aba1740f6fd217f7722f2f4c02888/frontend/app/src/components/history/transactions/TransactionContent.vue#L500

Reply:
Yeah, my bad